source/declarations.tex

```diff
@@ -7161,13 +7156,23 @@
 
 \begin{bnf}
 \nontermdef{using-enum-declaration}\br
-    \terminal{using} elaborated-enum-specifier \terminal{;}
+    \terminal{using enum} using-enum-declarator \terminal{;}
```

Comment:
@jensmaurer Would you rather use `\keyword{using} \keyword{enum}` here?

Reply:
Yes; also on the line prior to that.

Reply:
You mean in the heading?

Reply:
No. I was confused (I thought the red stuff was still there). Ignore this, please.